sal_interfaces/CCCamera/CCCamera_Events.xml

```diff
@@ -17,7 +17,7 @@
             <Enumeration>OFFLINE_AVAILABLE,OFFLINE_PUBLISH_ONLY</Enumeration>
         </item>
         <item>
-            <EFDB_Name>transitionTimestamp</EFDB_Name>
+            <EFDB_Name>transitionTimestA</EFDB_Name>
```

Comment:
Why are we abbreviating timestamp to timestA here and below? Is there some length constraint in Java?

Reply:
This was also fixed by the missing 3 commits, and was a mistake introduced by replacing Amp with A (which also replaced timestamp with timestA)